{
  "about": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-009008.html",
  "title": "python-cryptography における入力確認に関する脆弱性",
  "link": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-009008.html",
  "description": "python-cryptography には、入力確認に関する脆弱性が存在します。",
  "identifier": "JVNDB-2018-009008",
  "references": [
    {
      "text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-10903",
      "source": "CVE",
      "id": "CVE-2018-10903"
    },
    {
      "text": "https://nvd.nist.gov/vuln/detail/CVE-2018-10903",
      "source": "NVD",
      "id": "CVE-2018-10903"
    },
    {
      "text": "https://jvndb.jvn.jp/ja/cwe/CWE-20.html",
      "id": "CWE-20",
      "title": "不適切な入力確認(CWE-20)"
    }
  ],
  "cpe": [
    {
      "text": "cpe:/a:cryptography.io:python-cryptography",
      "version": "2.2",
      "vendor": "Cryptography",
      "product": "python-cryptography"
    },
    {
      "text": "cpe:/a:redhat:openstack",
      "version": "2.2",
      "vendor": "レッドハット",
      "product": "Red Hat OpenStack"
    },
    {
      "text": "cpe:/o:canonical:ubuntu_linux",
      "version": "2.2",
      "vendor": "Canonical",
      "product": "Ubuntu"
    }
  ],
  "cvss": [
    {
      "version": "2.0",
      "score": "5.0",
      "type": "Base",
      "severity": "Medium",
      "vector": "AV:N/AC:L/Au:N/C:P/I:N/A:N"
    },
    {
      "version": "3.0",
      "score": "7.5",
      "type": "Base",
      "severity": "High",
      "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
    }
  ],
  "date": "2018-11-05T18:11+09:00",
  "issued": "2018-11-05T18:11+09:00",
  "modified": "2018-11-05T18:11+09:00"
}
